hash/nes.xml

```diff
@@ -62224,6 +62208,31 @@ preliminary proto for the PAL version, still running on NTSC systems) or the gfx
 		</part>
 	</software>
 
+<!-- from cah4e3 -->
+	<software name="drmario2">
+		<description>Fang Kuai Zong Dong Yuan (Chi)</description>
```

Comment:
Does the published actually use “Chi” as an abbreviation for China?  If not, please don’t use it.  Use ISO-3166-1 Alpha-2 codes for country abbreviations if the manufacturer/publisher doesn’t have a consistent scheme.

Also, don’t use this style of spacing for Hanyu Pinyin.  Call it “Fāngkuài Zǒngdòngyuán” or something.

Reply:
Unabbreviated and suggested pinyin added.